README.md

```diff
@@ -1,59 +1,168 @@
-# TODO.md
+# Description
+[EN] Repositorie about how to do TO-DO list on markdown.
+
+# Content EN
+- [Structure of directories](#Structure-of-directories)
+- [What is TODO.md?](#What-is-`TODO.md`?)
+- [Why keep a TODO.md?](#Why-keep-a-`TODO.md`?)
+- [Why not using online management tools?](#Why-not-using-online-management-tools?)
+- [TODO.md format](#`TODO.md`-format)
+- [Examples](#examples)
+- [See also](#See-also)
+
+# Contenido ES
```

Comment:
Awesome idea for translation. Please keep the existing README.md and duplicate it to README_ES.md for ES language.